internal/configs/config_bootloaders.go

```diff
@@ -70,39 +73,33 @@ func Set_Cmdline(gpu_IDs []string) {
 	fileio.AppendContent(fmt.Sprintf(" vfio_pci.ids=%s", strings.Join(gpu_IDs, ",")), config.Path.CMDLINE)
 }
 
-// Configures systemd-boot using kernelstub
-func Set_KernelStub() string {
+// Set_KernelStub configures systemd-boot using kernelstub.
+func Set_KernelStub(isRoot bool) {
 	// Get the config
 	config := GetConfig()
 
 	// Get the kernel args
 	kernel_args := fileio.ReadFile(config.Path.CMDLINE)
 
-	// Write to logger
-	logger.Printf("Running command:\nsudo kernelstub -a \"%s\"\n", kernel_args)
-
-	// Run the command
-	_, err := command.Run("sudo", "kernelstub", "-a", kernel_args)
-	errorcheck.ErrorCheck(err, "Error, kernelstub command returned exit code 1")
-
-	// Return what we did
-	return fmt.Sprintf("Executed: sudo kernelstub -a \"%s\"", kernel_args)
+	// Run and log, check for errors
+	common.ErrorCheck(command.ExecAndLogSudo(isRoot, true,
+		"kernelstub -a "+kernel_args,
```

Comment:
`kernel_args` must be quoted,
![image](https://github.com/HikariKnight/quickpassthrough/assets/2557889/99cd8b67-5de8-4e8f-8591-e054b8ccb4d4)

so the output should have been
`sudo kernelstub -a "iommu=pt amd_iommu=on vfio_pci.ids=1af4:1050  vfio_pci.disable_vga=1"`

so we must change it to `'kernelstub -a "'+kernel_args+'"'`